cypress/testcases/hosts.spec.ts

```diff
@@ -10,8 +12,78 @@ const login = new LoginPage();
  */
 export function insertCustomYAML() {}
 it('should insert custom name into YAML', () => {
-    cy.login();
+    login.login();
```

Comment:
Should this be login.login() or cy.login()? It looks like you're using cy.login() everywhere else and that login was added to the commands for cypress.

Reply:
Done